dev/source/docs/style-guide.rst

```diff
@@ -421,6 +421,141 @@ Functions or variables that represent a value relative to a frame should be suff
     float course_deg;
 
 
+Abbreviations
```

Comment:
Let's make the title something like, "Preferred Units" or "variable suffix for units"

Reply:
Preferred Abbreviations for Units